CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 4.0.4 (unreleased)
 
+NOTES:
+
+* all: This Go module has been updated for deprecations in terraform-plugin-framework version 0.15.0
```

Comment:
Does this need to be called out for practitioners using the provider? 😉

Reply:
Have removed.